### DISCUSSION TITLE
Static Fuction

### DISCUSSION THREAD

**Discussion by andifahruddinakas** [esyede/rakit#21] [Bug] [answered]
https://github.com/esyede/rakit/blob/97414760bd298cdce0ba1bfc026ab8247542dfaa/system/request.php#L283

`return (static::ajax() && ! static::pjax() && static::accept_any()) || static::wants_json();`

karena ini : 
```php
public static function accept_any()
{
    $accept = static::accept();
    return count($accept) === 0 || (isset($accept[0]) && ($accept[0] === '*/*' || $accept[0] === '*'));
}
```

**Comment by esyede** [comment, marked as answer]
Edit: fixed in 360497103e93f5ec1ad6a348e8d1a918c44a6aa6 terimakasih!
